src/Core/Maintenance/System/Service/ShopConfiguratorDecorator.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Swag\LanguagePack\Core\Maintenance\System\Service;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\DataAbstractionLayer\Doctrine\RetryableTransaction;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Maintenance\System\Service\ShopConfigurator;
+
+class ShopConfiguratorDecorator extends ShopConfigurator
```

Comment:
Can you write an integration test please? Would be good to get an alert if the core service changes and this does not work anymore